<!DOCTYPE HTML>
<html lang="en">


<head>
    <meta charset="utf-8">
    <meta name="keywords" content="PROJECTS, Xie Lab; Tinghua University">
    <meta name="description" content="PROJECTS | Xie Lab">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
    <meta name="renderer" content="webkit|ie-stand|ie-comp">
    <meta name="mobile-web-app-capable" content="yes">
    <meta name="format-detection" content="telephone=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
    <title>PROJECTS | Xie Lab</title>
    <link rel="icon" type="image/png" href="/favicon.png">

    <link rel="stylesheet" type="text/css" href="/libs/awesome/css/all.css">
    <link rel="stylesheet" type="text/css" href="/libs/materialize/materialize.min.css">
    <link rel="stylesheet" type="text/css" href="/libs/aos/aos.css">
    <link rel="stylesheet" type="text/css" href="/libs/animate/animate.min.css">
    <link rel="stylesheet" type="text/css" href="/libs/lightGallery/css/lightgallery.min.css">
    <link rel="stylesheet" type="text/css" href="/css/matery.css">
    <link rel="stylesheet" type="text/css" href="/css/my.css">
    
    <script src="/libs/jquery/jquery.min.js"></script>
    
</head>


<body>
    <header class="navbar-fixed">
    <nav id="headNav" class="bg-color nav-transparent">
        <div id="navContainer" class="nav-wrapper head-container">
            <div class="brand-logo">
                <a href="/" class="waves-effect waves-light">
                    
                    <img src="/medias/logo.png" class="logo-img" alt="LOGO">
                    
                    <span class="logo-span">Xie Lab</span>
                </a>
            </div>
            

<a href="#" data-target="mobile-nav" class="sidenav-trigger button-collapse"><i class="fas fa-bars"></i></a>
<ul class="right nav-menu">
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/" class="waves-effect waves-light">
      
      <i class="fas fa-home" style="zoom: 0.6;"></i>
      
      <span>Home</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/news" class="waves-effect waves-light">
      
      <i class="far fa-newspaper" style="zoom: 0.6;"></i>
      
      <span>News</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/projects" class="waves-effect waves-light">
      
      <i class="fas fa-tasks" style="zoom: 0.6;"></i>
      
      <span>Projects</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/publications" class="waves-effect waves-light">
      
      <i class="fas fa-print" style="zoom: 0.6;"></i>
      
      <span>Publications</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/members" class="waves-effect waves-light">
      
      <i class="fas fa-user-graduate" style="zoom: 0.6;"></i>
      
      <span>Members</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/gallery" class="waves-effect waves-light">
      
      <i class="fas fa-photo-video" style="zoom: 0.6;"></i>
      
      <span>Gallery</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/intranet" class="waves-effect waves-light">
      
      <i class="fas fa-archive" style="zoom: 0.6;"></i>
      
      <span>Intranet</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/contact" class="waves-effect waves-light">
      
      <i class="fas fa-comments" style="zoom: 0.6;"></i>
      
      <span>Contact</span>
    </a>
    
  </li>
  
  <li>
    <a href="#searchModal" class="modal-trigger waves-effect waves-light">
      <i id="searchIcon" class="fas fa-search" title="Search" style="zoom: 0.85;"></i>
    </a>
  </li>
</ul>

<div id="mobile-nav" class="side-nav sidenav">

    <div class="mobile-head bg-color">
        
        <img src="/medias/logo.png" class="logo-img circle responsive-img">
        
        <div class="logo-name">Xie Lab</div>
        <div class="logo-desc">
            
            This is the official website of xie lab
            
        </div>
    </div>

    

    <ul class="menu-list mobile-menu-list">
        
        <li class="m-nav-item">
	  
		<a href="/" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-home"></i>
			
			Home
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/news" class="waves-effect waves-light">
			
			    <i class="fa-fw far fa-newspaper"></i>
			
			News
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/projects" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-tasks"></i>
			
			Projects
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/publications" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-print"></i>
			
			Publications
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/members" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-user-graduate"></i>
			
			Members
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/gallery" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-photo-video"></i>
			
			Gallery
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/intranet" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-archive"></i>
			
			Intranet
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/contact" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-comments"></i>
			
			Contact
		</a>
          
        </li>
        
        
    </ul>
</div>

        </div>

        
    </nav>

</header>

    
<script src="/libs/cryptojs/crypto-js.min.js"></script>
<script>
    (function() {
        let pwd = '';
        if (pwd && pwd.length > 0) {
            if (pwd !== CryptoJS.SHA256(prompt('输入密码——非实验室人员无法访问，请向管理员索要密码')).toString(CryptoJS.enc.Hex)) {
                alert('密码错误，将返回主页！');
                location.href = '/';
            }
        }
    })();
</script>




<div class="bg-cover pd-header post-cover" style="background-image: url('/medias/featureimages/14.jpg')">
    <div class="container" style="right: 0px;left: 0px;">
        <div class="row">
            <div class="col s12 m12 l12">
                <div class="brand">
                    <div class="description center-align post-title">
                        PROJECTS
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>




<main class="container content">

    
    <!-- 文章内容详情 -->
<div id="artDetail">
    <div class="card">
        <div class="card-content article-info">
            <div class="row tag-cate">
                <div class="col s7">
                    
                    <div class="article-tag">
                        
                            <a href="/">
                                <span class="chip bg-color"></span>
                            </a>
                        
                    </div>
                    
                </div>
                <div class="col s5 right-align">
                    
                </div>
            </div>

            <div class="post-info">
                

                
                <div class="post-date info-break-policy">
                    <i class="far fa-calendar-check fa-fw"></i>Update Date:&nbsp;&nbsp;
                    2019-12-21
                </div>
                

                

                
				
                
            </div>
            
        </div>
        <hr class="clearfix">
        <div class="card-content article-card-content">
            <div id="articleContent">
                <p><font size="4">The Xie lab started at Tsinghua University in Beijing since 2013. <font color="red"><strong>Our research centers on one key and fundamental question – how is the epigenome inherited and reprogrammed from parents to the next generation in mammals?</strong></font> One of the best systems to answer these questions is early development, when parental epigenetic information is being passed on to the progenies and when the first cell fate commitment events take place which subsequently determine all future body plans. However, the epigenetic reprogramming, especially at the level of chromatin, remains poorly understood given the extremely scarce experimental materials. We are dedicated to meet these challenges by first developing a series of ultra-low-input epigenetic analysis technologies, which are then applied to solve biological questions in combination with genetic tools. Our specific questions are described below: </font></p>
<h4 id="1-How-are-the-epigenomes-established-and-regulated-in-gametes"><a href="#1-How-are-the-epigenomes-established-and-regulated-in-gametes" class="headerlink" title="1. How are the epigenomes established and regulated in gametes?"></a>1. How are the epigenomes established and regulated in gametes?</h4><p>Using ultra-sensitive ChIP-seq and CUT&amp;RUN, we revealed landscapes of histone modifications in oocyte and sperm in zebrafish (<strong><em>Mol. Cell</em>, 2018</strong>), mouse (<strong><em>Nature</em>, 2016b; <em>Molecular Cell</em>, 2016a; <em>Nature Genetics</em>, 2019</strong>), and human (<strong><em>Science</em> 2019</strong>). Interestingly, histone modifications can adopt non-canonical forms in oocytes and possess functions that are distinct from those in somatic cells (<strong><em>Nature</em>, 2016a</strong>). </p>
<p>In addition, we also examined chromatin structure in 3D space. By successfully developing an ultrasensitive tool to study 3D chromatin organization (sis-HiC), we showed how chromatin is folded during mammalian gametogenesis (<strong><em>Mol. Cell</em>, 2019a; <em>Mol. Cell</em>, 2019b</strong>). </p>
<p>We also revealed that genomic imprints, the parental epigenetic memories that are essential for development, are established through SETD2-mediated histone modification-DNA methylation crosstalks in oocytes (<strong><em>Nature Genetics</em>, 2019).</strong> These marks are rather stable afterwards before they are removed again in primordial germ cells, where we found and how this is executed by a simple isoform switch of an epigenetic regulator TET1 <strong>(<em>Molecular Cell</em>, 2016b)</strong>.<br><img src="/projects/index/figure1.jpg" style="zoom:100%;"></p>
<h4 id="2-How-are-the-epigenomes-reprogrammed-during-parental-to-zygotic-transition"><a href="#2-How-are-the-epigenomes-reprogrammed-during-parental-to-zygotic-transition" class="headerlink" title="2. How are the epigenomes reprogrammed during parental-to-zygotic transition?"></a>2. How are the epigenomes reprogrammed during parental-to-zygotic transition?</h4><p>By doing so, we are among the first to provide a global view of chromatin accessible states in preimplantation embryos at the DNA level in human and mouse <strong>(<em>Nature</em>, 2016a; <em>Nature</em>, 2018)</strong>. Furthermore, we sought to determine whether histone modifications from each parent can be inherited to early embryos in mouse <strong>(<em>Nature</em>, 2016b; <em>Molecular Cell</em>, 2016a; <em>Nature Genetics</em>, 2019a),</strong> zebrafish <strong>(<em>Mol. Cell</em>, 2018)</strong> and human <strong>(<em>Science</em> 2019)</strong>. Finally, we showed how chromatin folds in 3D spaces after fertilization <strong>(<em>Nature</em>, 2017)</strong>.  </p>
<img src="/projects/index/figure2.jpg" style="zoom:100%;">

<h4 id="3-How-are-the-embryonic-epigenomes-established-and-regulated"><a href="#3-How-are-the-embryonic-epigenomes-established-and-regulated" class="headerlink" title="3. How are the embryonic epigenomes established and regulated?"></a>3. How are the embryonic epigenomes established and regulated?</h4><p>Given epigenetic modifications are extensively removed upon fertilization, one critical question is how the embryonic epigenome is established afterwards. As exploratory studies, we examined how DNA methylomes (<strong><em>Nature Genetics</em>, 2018</strong>) and histone modifications (<strong><em>Nature Genetics</em>, 2019</strong>) are established during about 10 early mouse lineage specification during peri- and postimplantation development (E3.5 to E7.5). In addition, we also investigate the reprogramming of chromatin molecular architecture during postimplantation development <strong>(<em>Nature Genetics</em>, 2018; <em>Nature Genetics</em>, 2019b)</strong>.</p>
<img src="/projects/index/figure3.jpg" style="zoom:100%;">

<h4 id="4-How-the-epigenomes-are-regulated-in-embryonic-stem-cells"><a href="#4-How-the-epigenomes-are-regulated-in-embryonic-stem-cells" class="headerlink" title="4. How the epigenomes are regulated in embryonic stem cells?"></a>4. How the epigenomes are regulated in embryonic stem cells?</h4><p>Besides early development, we also use embryonic stem cells to decipher mechanistic insights on epigenetic regulation of transcription. For example, we showed how developmental genes manage to avoid DNA methylation regardless whether they are transcribed or not (<strong><em>Genome Biology</em>, 2018</strong>). </p>
<p>For details, please refer to our reviews in chromatin reprogramming in gametogenesis and early development (<strong><em>Trends in Cell Biology</em>, 2018</strong>; <strong><em>Nature Reviews in Molecular Cell Biology</em>, 2019</strong>)    </p>
 <img src="/projects/index/figure4.jpg" style="zoom:100%;">

<hr>
<p><br><br></p>
<p><font color="red" font size="4"><strong>Specifically, our findings are described in detail below.</strong></font><br><br><br></p>
<h4 id="1-We-are-among-the-first-to-demonstrate-how-3D-chromatin-structure-is-reprogrammed-in-early-mammalian-development-Nature-2017"><a href="#1-We-are-among-the-first-to-demonstrate-how-3D-chromatin-structure-is-reprogrammed-in-early-mammalian-development-Nature-2017" class="headerlink" title="1. We are among the first to demonstrate how 3D chromatin structure is reprogrammed in early mammalian development (Nature, 2017)"></a>1. We are among the first to demonstrate how 3D chromatin structure is reprogrammed in early mammalian development (Nature, 2017)</h4><p>In eukaryotes, the genetic material DNA does not exist in a linear manner, but instead exists in the nucleus in a certain three-dimensional structure. The 3D chromatin structure plays an important role in biological processes such as gene expression regulation, DNA replication and cell division. In mammals, chromatin organization undergoes drastic reprogramming after fertilization. However, the three-dimensional structure of chromatin and its reprogramming in preimplantation development remain poorly understood. Here, by developing a low-input Hi-C (genome-wide chromosome conformation capture) approach, we examined the reprogramming of chromatin organization during early development in mice (Du et al., 2017) (Fig. 1). We found that oocytes in metaphase II show homogeneous chromatin folding that lacks detectable topologically associating domains (TADs) and chromatin compartments. Strikingly, chromatin showed greatly diminished higher-order structure after fertilization. Unexpectedly, the subsequent establishment of chromatin organization is a prolonged process that extends through preimplantation development, as characterized by slow consolidation of TADs and segregation of chromatin compartments. The two sets of parental chromosomes are spatially separated from each other and display distinct compartmentalization in zygotes. Such allele separation and allelic compartmentalization can be found as late as the 8-cell stage. Finally, we showed that chromatin compaction in preimplantation embryos can partially proceed in the absence of zygotic transcription and is a multi-level hierarchical process. <strong>Taken together, our data suggest that chromatin may exist in a markedly relaxed state after fertilization, followed by progressive maturation of higher-order chromatin architecture during early development.</strong> <strong>This work was published in *Nature* in June 2017 and</strong> was cited for <strong>22</strong> times thus far.</p>
<h4 id="2-We-demonstrated-how-chromatin-is-reprogrammed-at-the-DNA-levels-in-human-early-embryos-Nature-2018"><a href="#2-We-demonstrated-how-chromatin-is-reprogrammed-at-the-DNA-levels-in-human-early-embryos-Nature-2018" class="headerlink" title="2. We demonstrated how chromatin is reprogrammed at the DNA levels in human early embryos (Nature, 2018)."></a>2. We demonstrated how chromatin is reprogrammed at the DNA levels in human early embryos (Nature, 2018).</h4><p>The state of chromatin dictates fundamental cellular processes including gene expression. Accessible chromatin can expose key DNA sequences to transcription regulators leading to gene activation. As a result, accessible chromatin often marks regulatory sequences such as promoters and enhancers. ATAC-seq is a highly sensitive approach to detect open chromatin (Buenrostro et al., 2013; Buenrostro et al., 2015). By developing an improved assay for transposase-accessible chromatin with high-throughput sequencing (mini-ATAC-seq), we investigate chromatin states in human preimplantation development (Wu et al., 2018) (Fig. 2). We found widespread accessible chromatin regions in early human embryos that overlap extensively with putative cis-regulatory sequences and transposable elements. Integrative analyses show both conservation and divergence in regulatory circuitry between human and mouse early development, and between human pluripotency in vivo and human embryonic stem cells. In addition, we find widespread open chromatin regions before zygotic genome activation (ZGA). The accessible chromatin loci are readily found at CpG-rich promoters. Unexpectedly, many others reside in distal regions that overlap with DNA hypomethylated domains in human oocytes and are enriched for transcription factor-binding sites. A large portion of these regions then become inaccessible after ZGA in a transcription-dependent manner. Notably, such extensive chromatin reorganization during ZGA is conserved in mice and correlates with the reprogramming of the non-canonical histone mark H3K4me3, which is uniquely linked to genome silencing. <strong>Taken together, these data not only reveal a conserved principle that underlies the chromatin transition during mammalian ZGA, but also help to advance our understanding of epigenetic reprogramming during human early development and in vitro fertilization.</strong> <strong>This work was published in *Nature* in May 2018.</strong> </p>
<h4 id="3-We-demonstrated-how-the-zygotic-epigenome-is-established-during-early-embryonic-lineage-determination-in-mammals-Nature-Genetics-2018"><a href="#3-We-demonstrated-how-the-zygotic-epigenome-is-established-during-early-embryonic-lineage-determination-in-mammals-Nature-Genetics-2018" class="headerlink" title="3. We demonstrated how the zygotic epigenome is established during early embryonic lineage determination in mammals (Nature Genetics, 2018)."></a>3. We demonstrated how the zygotic epigenome is established during early embryonic lineage determination in mammals (Nature Genetics, 2018).</h4><p>In mammals, early lineage specification in preimplantation and postimplantation embryonic development generates founder tissues for all subsequent somatic development. The first lineage specification starts at the morula stage, when the inner cell mass (ICM) and the trophectoderm (TE) begin to segregate. However, because of the limited materials available and the difficulty of tissue isolation in early embryos, lineage-specific regulation of transcriptomes and epigenomes in peri- and postimplantation embryos is poorly characterized. By developing a low-input method for genome-wide DNA-methylation profiling STEM-seq (small-scale TELP-enabled methylome sequencing), we examined the DNA methylation and chromatin structure in 11 lineage tissues in peri- and postimplantation mouse embryos (E3.5 to E7.5)(Zhang et al., 2018). We reported a comprehensive investigation of transcriptomes and base-resolution methylomes for early lineages in peri- and postimplantation mouse embryos. We found allele-specific and lineage-specific de novo methylation at CG and CH sites that led to differential methylation between embryonic and extraembryonic lineages at promoters of lineage regulators, gene bodies, and DNA-methylation valleys. By using Hi-C experiments to define chromatin architecture across the same developmental period, we demonstrated that both global demethylation and remethylation in early development correlate with chromatin compartments. Dynamic local methylation was evident during gastrulation, which enabled the identification of putative regulatory elements. Finally, we found that de novo methylation patterning does not strictly require implantation. <strong>These data revealed dynamic transcriptomes, DNA methylomes, and 3D chromatin landscapes during the earliest stages of mammalian lineage specification (Fig. 3). This work was published in <em>Nature Genetics</em> in January 2018.</strong> </p>
<h4 id="4-We-demonstrated-how-the-DNA-methylation-valleys-at-promoters-of-developmental-genes-are-regulated-by-Polycomb-protein-Genome-Biology-2018"><a href="#4-We-demonstrated-how-the-DNA-methylation-valleys-at-promoters-of-developmental-genes-are-regulated-by-Polycomb-protein-Genome-Biology-2018" class="headerlink" title="4. We demonstrated how the DNA methylation valleys at promoters of developmental genes are regulated by Polycomb protein (Genome Biology, 2018)."></a>4. We demonstrated how the DNA methylation valleys at promoters of developmental genes are regulated by Polycomb protein (Genome Biology, 2018).</h4><p>DNA methylation is a major epigenetic mark regulating gene repression, genomic imprinting, and X chromosome inactivation. Intriguingly, DNA methylation is constantly absent at many promoters of key regulators for embryonic development regardless of their transcription activities. These promoters frequently show unusually large hypomethylated regions extending well beyond their proximal promoters, a pattern that we termed DNA methylation valley (DMV). However, it remains elusive how most developmental genes can evade DNA methylation regardless of their transcriptional states. Therefore, we sought to determine the molecular mechanisms underlying the regulation of hypomethylation in DMVs (Li et al., 2018). We showed that DMVs are hypomethylated in development and are highly conserved across vertebrates. Importantly, DMVs are hotspots of regulatory regions for key developmental genes and show low levels of deamination mutation rates. By analyzing a panel of DNA methylomes from mouse tissues, we identify a subset of DMVs that are dynamically methylated. These DMVs are strongly enriched for Polycomb-deposited H3K27me3 when the associated genes are silenced, and surprisingly showed elevated DNA methylation upon gene activation. 4C-seq analyses indicates that Polycomb-bound DMVs form insulated and self-interacting chromatin domains. Further investigations showed that DNA hypomethylation is better correlated with the binding of Polycomb than with H3K27me3. Indeed, we observed aberrant methylation in DMVs in mouse embryonic stem cells deficient in the EED protein. Finally, we showed that Polycomb regulates hypomethylation of DMVs likely through ten-eleven translocation (TET) proteins (<strong>Fig. 4</strong>). <strong>This work was published in <em>Genome Biology</em> in February 2018.</strong></p>
<p>Taken together, our recent findings revealed how the epigenome is reprogrammed and inherited from parents to early embryos in mammals. With several cutting-edge technologies newly developed in lab, we are poised to further investigate the mechanisms and functions underlying these fascinating processes. Importantly, we believe this is only the beginning to unravel many mysteries of early development.</p>

            </div>
            <hr/>

            



            <div class="tag_share" style="display: block;">
                <div class="post-meta__tag-list" style="display: inline-block;">
                    
                        <div class="article-tag">
                            
                                <a href="/">
                                    <span class="chip bg-color"></span>
                                </a>
                            
                        </div>
                    
                </div>
                <div class="post_share" style="zoom: 80%; width: fit-content; display: inline-block; float: right; margin: -0.15rem 0;">
                    <link rel="stylesheet" type="text/css" href="/libs/share/css/share.min.css">

<div id="article-share">
    
    

    

</div>

                </div>
            </div>
            
        </div>
    </div>

    

    

    

    

    

    

<article id="prenext-posts" class="prev-next articles">
    <div class="row article-row">
        
        <div class="article col s12 m6" data-aos="fade-up" data-aos="fade-up">
            <div class="article-badge left-badge text-color">
                <i class="far fa-dot-circle"></i>&nbsp;Current
            </div>
            <div class="card">
                <a href="/projects/index.html">
                    <div class="card-image">
                        
                        
                        <img src="/medias/featureimages/14.jpg" class="responsive-img" alt="PROJECTS">
                        
                        <span class="card-title">PROJECTS</span>
                    </div>
                </a>
                <div class="card-content article-content">
                    <div class="summary block-with-text">
                        
                            These are our projects recently.
                        
                    </div>
                    <div class="publish-info">
                            <span class="publish-date">
                                <i class="far fa-clock fa-fw icon-date"></i>2019-11-06
                            </span>
                        <span class="publish-author">
                            
                            <i class="fas fa-user fa-fw"></i>
                            Xie Lab
                            
                        </span>
                    </div>
                </div>

                
                <div class="card-action article-tags">
                    
                    <a href="/">
                        <span class="chip bg-color"></span>
                    </a>
                    
                </div>
                
            </div>
        </div>
        
        
        <div class="article col s12 m6" data-aos="fade-up">
            <div class="article-badge right-badge text-color">
                Current&nbsp;<i class="far fa-dot-circle"></i>
            </div>
            <div class="card">
                <a href="/projects/index.html">
                    <div class="card-image">
                        
                        
                        <img src="/medias/featureimages/14.jpg" class="responsive-img" alt="PROJECTS">
                        
                        <span class="card-title">PROJECTS</span>
                    </div>
                </a>
                <div class="card-content article-content">
                    <div class="summary block-with-text">
                        
                            These are our projects recently.
                        
                    </div>
                    <div class="publish-info">
                            <span class="publish-date">
                                <i class="far fa-clock fa-fw icon-date"></i>2019-11-06
                            </span>
                        <span class="publish-author">
                            
                            <i class="fas fa-user fa-fw"></i>
                            Xie Lab
                            
                        </span>
                    </div>
                </div>

                
                <div class="card-action article-tags">
                    
                    <a href="/">
                        <span class="chip bg-color"></span>
                    </a>
                    
                </div>
                
            </div>
        </div>
        
    </div>
</article>

</div>


<script>
    $('#articleContent').on('copy', function (e) {
        // IE8 or earlier browser is 'undefined'
        if (typeof window.getSelection === 'undefined') return;

        var selection = window.getSelection();
        // if the selection is short let's not annoy our users.
        if (('' + selection).length < Number.parseInt('50')) {
            return;
        }

        // create a div outside of the visible area and fill it with the selected text.
        var bodyElement = document.getElementsByTagName('body')[0];
        var newdiv = document.createElement('div');
        newdiv.style.position = 'absolute';
        newdiv.style.left = '-99999px';
        bodyElement.appendChild(newdiv);
        newdiv.appendChild(selection.getRangeAt(0).cloneContents());

        // we need a <pre> tag workaround.
        // otherwise the text inside "pre" loses all the line breaks!
        if (selection.getRangeAt(0).commonAncestorContainer.nodeName === 'PRE') {
            newdiv.innerHTML = "<pre>" + newdiv.innerHTML + "</pre>";
        }

        var url = document.location.href;
        newdiv.innerHTML += '<br />'
            + 'From: Xie Lab<br />'
            + 'Author: Xie Lab<br />'
            + 'Link: <a href="' + url + '">' + url + '</a><br />'
            + '本文著作权归清华大学生命科学学院颉伟实验室所有，任何形式的转载都请注明出处。';

        selection.selectAllChildren(newdiv);
        window.setTimeout(function () {bodyElement.removeChild(newdiv);}, 200);
    });
</script>


<!-- 代码块功能依赖 -->
<script type="text/javascript" src="/libs/codeBlock/codeBlockFuction.js"></script>

<!-- 代码语言 -->

<script type="text/javascript" src="/libs/codeBlock/codeLang.js"></script>

    
<!-- 代码块复制 -->

<script type="text/javascript" src="/libs/codeBlock/codeCopy.js"></script>


<!-- 代码块收缩 -->

<script type="text/javascript" src="/libs/codeBlock/codeShrink.js"></script>


<!-- 代码块折行 -->

    

</main>



    <footer class="page-footer bg-color">
    <div class="container row center-align">
        <div class="col s12 m8 l8 copy-right">
            Copyright&nbsp;&copy;
            <span id="year">2019</span>
            <a href="https://github.com/xielabtsinghua" target="_blank">Xie Lab</a>
            |&nbsp;Built by&nbsp;<a href="https://hermanzhaozzzz.github.io/" target="_blank">Huanan Zhao</a>
            <br>
            
            
            
            
            
            
            <span id="busuanzi_container_site_pv">
                |&nbsp;<i class="far fa-eye"></i>总访问量:&nbsp;<span id="busuanzi_value_site_pv"
                    class="white-color"></span>&nbsp;次
            </span>
            
            
            <span id="busuanzi_container_site_uv">
                |&nbsp;<i class="fas fa-users"></i>&nbsp;总访问人数:&nbsp;<span id="busuanzi_value_site_uv"
                    class="white-color"></span>&nbsp;人
            </span>
            
            <br>
            
            <br>
            
        </div>
        <div class="col s12 m4 l4 social-link social-statis">














</div>
    </div>
</footer>

<div class="progress-bar"></div>


    <!-- 搜索遮罩框 -->
<div id="searchModal" class="modal">
    <div class="modal-content">
        <div class="search-header">
            <span class="title"><i class="fas fa-search"></i>&nbsp;&nbsp;Search</span>
            <input type="search" id="searchInput" name="s" placeholder="Please enter a search keyword"
                   class="search-input">
        </div>
        <div id="searchResult"></div>
    </div>
</div>

<script src="/js/search.js"></script>
<script type="text/javascript">
$(function () {
    searchFunc("/" + "search.xml", 'searchInput', 'searchResult');
});
</script>
    <!-- 回到顶部按钮 -->
<div id="backTop" class="top-scroll">
    <a class="btn-floating btn-large waves-effect waves-light" href="#!">
        <i class="fas fa-arrow-up"></i>
    </a>
</div>


    <script src="/libs/materialize/materialize.min.js"></script>
    <script src="/libs/masonry/masonry.pkgd.min.js"></script>
    <script src="/libs/aos/aos.js"></script>
    <script src="/libs/scrollprogress/scrollProgress.min.js"></script>
    <script src="/libs/lightGallery/js/lightgallery-all.min.js"></script>
    <script src="/js/matery.js"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->


    <!-- Baidu Analytics -->

    <!-- Baidu Push -->

<script>
    (function () {
        var bp = document.createElement('script');
        var curProtocol = window.location.protocol.split(':')[0];
        if (curProtocol === 'https') {
            bp.src = 'https://zz.bdstatic.com/linksubmit/push.js';
        } else {
            bp.src = 'http://push.zhanzhang.baidu.com/push.js';
        }
        var s = document.getElementsByTagName("script")[0];
        s.parentNode.insertBefore(bp, s);
    })();
</script>

    
    
    <script async src="/libs/others/busuanzi.pure.mini.js"></script>
    

    

    

    

    

    
    
    
    <script src="/libs/instantpage/instantpage.js" type="module"></script>
    

</body>

</html>
